ReflectionAnalyzers/ReflectionAnalyzers.csproj

```diff
@@ -20,18 +24,25 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>0.1.20.0-dev</Version>
+    <Version>0.1.20-dev</Version>
+    <DevelopmentDependency>true</DevelopmentDependency>
     <Description>Analyzers for System.Reflection</Description>
-    <PackageLicenseUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageLicenseUrl>
-    <RepositoryUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</RepositoryUrl>
-    <PackageTags>Reflection Roslyn Diagnostic Analyzer DotnetAnalyzers</PackageTags>
+    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
```

Comment:
⚠ I **do not** recommend using `<PackageLicenseUrl>` and it causes a pack warning due to being deprecated.

```suggestion
    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

https://docs.microsoft.com/en-us/nuget/reference/nuspec#license
https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target
https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/nuget#important-nuget-package-metadata